owncloudApp/build.gradle

```diff
@@ -60,19 +55,19 @@ tasks.withType(Test) {
 }
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 14
-        targetSdkVersion 26
+        minSdkVersion 15
```

Comment:
Let's keep this in 14, I know is a pretty old version but we will upgrade it after discussing it in https://github.com/owncloud/android/issues/2431

Reply:
Google removed 14 from any statistic, because is as rare like a unicorn.
It's even not worth to discuss to revert to 14, you will loose 0,0001 % of users
It tried to revert
<img width="984" alt="image" src="https://user-images.githubusercontent.com/3314607/52141601-87dd9700-2656-11e9-8a79-1c342c5cca15.png">

Let' go 15. Even 15 is like a unicorn